src/main/java/com.epam.tc.nitcenkov/hw7/forms/MetalsAndColorsForm.java

```diff
@@ -0,0 +1,74 @@
+package com.epam.tc.nitcenkov.hw7.forms;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.Checkbox;
+import com.epam.jdi.light.ui.html.elements.common.Icon;
+import com.epam.jdi.light.ui.html.elements.complex.MultiSelector;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.tc.nitcenkov.hw7.entities.MetalAndColors;
+import com.epam.tc.nitcenkov.hw7.util.DataParser;
+
+public class MetalsAndColorsForm extends Form<MetalAndColors> {
+
+    @UI("[name=custom_radio_odd]")
+    public RadioButtons oddButton;
+
+    @UI("[name=custom_radio_even]")
+    public RadioButtons evenButton;
+
+    @FindBy(id = "calculate-button")
+    public Button calculateButton;
+
+    @FindBy(css = "#elements-checklist input")
+    public Checklist elements;
+
+    @JDropdown(root = "#colors",
+               value = ".filter-option",
+               list = "li",
+               expand = ".caret")
+    public Dropdown color;
+
+    @JDropdown(root = "#metals",
+               value = ".filter-option",
+               list = "li",
+               expand = ".caret")
+    public Dropdown metals;
+
+    @JDropdown(root = "#vegetables",
+               value = "span.filter-option",
+               list = "li",
+               expand = ".caret")
+    public Dropdown vegetablesDropDown;
+
+    @FindBy(css = "#submit-button")
+    public Button submit;
+
+
+
+    public void fill(MetalAndColors metalAndColors) {
```

Comment:
Why don't you override only one submit(Entity e) method instead of two methods full(Entity e) and submit() ?